src/main/java/uk/gov/hmcts/reform/civil/handler/callback/user/RespondToWrittenRepresentationHandler.java

```diff
@@ -49,9 +57,19 @@ private CallbackResponse submitClaim(CallbackParams callbackParams) {
         String userId = idamClient.getUserInfo(callbackParams.getParams().get(BEARER_TOKEN).toString()).getUid();
         CaseData.CaseDataBuilder caseDataBuilder = caseData.toBuilder();
         String role = DocUploadUtils.getUserRole(caseData, userId);
+        List<Element<Document>> responseDocumentToBeAdded = caseData.getGeneralAppWrittenRepUpload();
```

Comment:
Suggestion: lines 60-63 can be replace with
List<Element> tobeAdded = Optional.ofNullable(caseData.getGeneralAppWrittenRepUpload()).orElse(new ArrayList<>());